about-the-apprentices/2022/Terri.md

```diff
@@ -10,4 +10,4 @@ One of my favorite things about programming is it's creative and like solving pu
 The Personal Computer
 
 * Desktop, laptop, or tablet? Which do you prefer to use?
-Desktop for sure!
+Desktop for sure!!
```

Comment:
Hey @tisidro, one note about the commit message here. The commit message "chore: Adding Terri file information" is very similar to the previous commit subject and doesn't speak to the reason for the changes here. In the future, I'd consider something like "chore: Update punctuation" and consider including a reason why there should be two exclamation points instead of one.

Reply:
Ah, this makes sense! Thanks for the explanation--it helps me so much at this point in my learning :)

Reply:
Jordan would you mind rebasing/merging when you get a chance? Thanks!

Reply:
Hey Terri, just a couple of things:

1. The reviewee, not the reviewer, is the one to rebase and squash a branch before pushing it up to the remote. Check out the steps starting at number five [here](https://github.com/sparkbox/standard/tree/main/code-style/git#the-sparkbox-git-flow).
2. Just to make sure that every apprentice has an opportunity to both submit a PR and review a PR, let's find an apprentice to do the merging. Since every apprentice is only reviewing and submitting one PR, there should be one person available to do this one.